### PR TITLE
Pull libsigar from GCS for Centos{6,7}

### PR DESCRIPTION
**Pull request by sambitesh**
We now pull down the libsigar artifacts (.targz) from GCS for Centos
instead of ivy.

Co-authored-by: Nandish Jayaram <njayaram@pivotal.io>
Co-authored-by: Sambitesh Dash <sdash@pivotal.io>

## Here are some reminders before you submit the pull request
- [x ] Add tests for the change
- [x ] Document changes
- [x ] Communicate in the mailing list if needed
- [x ] Pass `make installcheck`

### CODE REVIEW

File: concourse/pipelines/templates/gpdb-tpl.yml [hunk 1]
```diff
@@ -422,6 +422,13 @@ resources:
     json_key: ((gcs-key))
     regexp: centos6/libquicklz-devel-(\d.*)\.el6\.x86_64\.rpm
 
+- name: libsigar-centos6-gcs
```
Comment (bradfordboyle) [inline concourse/pipelines/templates/gpdb-tpl.yml]: Do we need the `-gcs` suffix? It doesn't look like we are using it w/ the other GCS resources (e.g., libquicklz-centos7). I vote we remove it.

File: concourse/scripts/sync_tools.bash [hunk 2]
```diff
@@ -38,9 +38,14 @@ function _main() {
         ;;
   esac
 
+  # We have moved out of ivy for Centos{6,7}, so make sync_tools
```
Comment (bradfordboyle) [inline concourse/scripts/sync_tools.bash]: w/ @dsharp-pivotal: Does `make sync_tools` still do anything on Centos? If it is effectively a no-op, we should remove the `sync_tools` task from the Centos compile jobs.

The `mkdir ...` command would then be moved into the compile task.

Comment (sambitesh) [reply concourse/scripts/sync_tools.bash]: `make sync_tools` also installs ORCA. So it is required for Centos{6,7}

**Comment by amilkh** [comment]
See below commits for fixes and repush:
4547b53820
91a9588e9b

**Pull request merged**